ui/packages/catalog-ui-search/src/main/webapp/js/model/QueryResponse.js

```diff
@@ -139,16 +139,21 @@ module.exports = Backbone.AssociatedModel.extend({
             handled = true
             res.options = options
             switch (res.code) {
-              case 400:
-              case 404:
-              case 500:
+              case -32000:
```

Comment:
This js file should be identical to the 2.13.x change, except that the formatting is different on `master`.